ceph.spec.in

```diff
@@ -1450,7 +1450,7 @@ mkdir -p %{buildroot}%{_localstatedir}/lib/ceph/bootstrap-rbd
 mkdir -p %{buildroot}%{_localstatedir}/lib/ceph/bootstrap-rbd-mirror
 
 # prometheus alerts
-install -m 644 -D monitoring/prometheus/alerts/ceph_default_alerts.yml %{buildroot}/etc/prometheus/ceph/ceph_default_alerts.yml
+install -m 644 -D monitoring/ceph-mixin/output/alerts/ceph_default_alerts.yml %{buildroot}/etc/prometheus/ceph/ceph_default_alerts.yml
```

Comment:
As it stands, all the current `ceph.spec.in` changeset does is break the `ceph-prometheus-alerts` subpackage.

Further down in the spec file there is a section that looks like this:

```
%files prometheus-alerts
%if 0%{?suse_version}
%attr(0755,root,root) %dir %{_sysconfdir}/prometheus
%endif
%attr(0755,root,root) %dir %{_sysconfdir}/prometheus/ceph
%config %{_sysconfdir}/prometheus/ceph/ceph_default_alerts.yml
```

You'll need to make this change there, too.

Reply:
If I understand correctly I don't need to update this part of the spec file because I'm doing the install at the exact same place. But maybe I'm wrong?